assets/src/helpers/client.ts

```diff
@@ -60,14 +60,12 @@ export function buildClient(gqlUrl, wsUrl, fetchToken) {
   })
 
   setInterval(() => {
-    if (socket.transport.readyState == WebSocket.CLOSED) {
+    if (!socket.conn) {
```

Comment:
are you sure this ever goes undefined?

Reply:
it happened every time i called socket.disconnect() when testing